.changes/unreleased/BUG FIXES-20250108-173554.yaml

```diff
@@ -0,0 +1,5 @@
+kind: BUG FIXES
+body: Updated dependency `github.com/hashicorp/go-slug` `v0.16.0` => `v0.16.3` to integrate latest changes.
```

Comment:
❓ Do we always refer to these kind of dependency upgrade as bug fixes, or did this upgrade infact fix a bug, besides integrating the latest changes.

Reply:
I had the same question, but then I found some dependency updates have been included in the changelog previously. The release notes for go-slug indicated that this updates would help prevent a panic from occurring so I figured it was user facing - @NodyHub could you please confirm?

Reply:
Background to this go-slug release is that we addressed certain security related issues in [`v0.16.3 `](https://github.com/hashicorp/go-slug/releases/tag/v0.16.3). The current release notes are not too much descriptive for now, bc/ we want to get the fix out asap and get our products fixed in the first place. After the updates are rolled out, we are creating a security bulletin and assign a CVE to the older versions `<v0.16.3`.

Does that make sense?

Reply:
That makes sense - in that case think this PR should wait for further details and be updated with the CVE name, as that's happened in past changelog entries. Thanks for the context!

Reply:
```suggestion
body: Updated dependency `github.com/hashicorp/go-slug` `v0.16.0` => `v0.16.3` to integrate latest changes (fix for CVE-2025-0377)
```

Reply:
Hey @SarahFrench , I checked in with @dduzgun-security on the CVE process and he reserved `CVE-2025-0377` for the bug. :)

Reply:
Thanks!